backend/app.session.ts

```diff
@@ -0,0 +1,24 @@
+import session from 'express-session';
+import {ServerConfiguration} from './app.config';
+import PostgresSession from './services/session';
+import db from './services/database';
+
+export interface UserSession {
+    id: number;
+}
+
+declare module 'express-session' {
+    interface SessionData {
+        user: UserSession;
+    }
+}
+
+export default session({
+    secret: ServerConfiguration.sessionSecret,
+    resave: false,
+    saveUninitialized: false,
+    cookie: { maxAge: 1000 * 60 * 60 * 24 * 7 },
```

Comment:
put a comment to know how much it makes in hour or day

Reply:
Fixed in https://github.com/csc-667-spring-2023-roberts/term-project-frenchies/pull/3/commits/a478fa53114858fec3541e4126fdd9e4b23cd530